app/javascript/packs/application.js

```diff
@@ -7,9 +7,11 @@ import Rails from "@rails/ujs"
 
 Rails.start()
 
-import 'uswds/src/stylesheets/uswds.scss'
+import 'css/application.scss'
```

Comment:
**issue:** When I run my Rails server locally I'm getting this error, even after restarting the server.
<img width="1022" alt="Screen Shot 2021-12-09 at 3 59 49 PM" src="https://user-images.githubusercontent.com/3675092/145494489-dc36d8c7-d1fd-4284-860f-f3aca697ec91.png">

**suggestion:** I was able to get this running by requiring it instead but not totally sure if that's the right approach?
```
require('css/application.scss')
```

Reply:
Hmm - I wonder if this is just a development/cacheing weirdness. Can you try reverting to the `import` version, and running `bundle exec rake assets:clobber` before trying to load the page again?

Reply:
Yes, I was able to recreate this behavior. Looks like it works fine as long as I remember to run `yarn install` _before_ loading the page on the new branch. If you load the page without running yarn install you get that compile error, but it doesn't get cleared until you either run `assets:clobber` or make a change to our JS. I think the `require` change worked, but also any other change would have triggered a recompilation and fixed the error.

Reply:
ah yep, thanks for digging into this!